web/components/TeamList.js

```diff
@@ -1,157 +1,44 @@
 import React from 'react';
-
```

Comment:
This coding standard _will_ be enforced. We want a single empty line separating components imported from node_modules and components imported from local source code